components/brave_new_tab_ui/containers/newTab/settings/sponsoredImagesToggle.tsx

```diff
@@ -25,94 +25,229 @@ interface Props {
   rewardsEnabled: boolean
   adsEnabled: boolean
   canSupportAds: boolean
+  isExternalWalletConnected: boolean
 }
 
-const Container = styled.div`
-  background-color: var(--info-background);
-  border-radius: 8px;
-  padding: 8px;
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  gap: 10px;
-`
-
 const ToggleRow = styled.div`
   width: 100%;
   display: flex;
   align-items: center;
   justify-content: space-between;
-  padding: 12px 14px;
-  border-radius: 10px;
+  padding: 8px 0px;
   background-color: var(--background1);
 `
 
-const DescriptionRow = styled.div`
-  width: 100%;
-  display: grid;
-  grid-template-rows: fit-content(16px) 1fr;
-  grid-template-columns: 16px 1fr;
-  padding: 0 8px;
-  gap: 8px;
-  color: #339AF0;
+const ControlsContainer = styled.span`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+`
+
+const InfoIcon = styled.span`
+  position: relative;
+  margin: 0px 18px;
+
+  > .icon {
+    height: 17px;
+    width: auto;
+    vertical-align: middle;
+  }
+
+  .tooltip {
+    position: absolute;
+    bottom: 100%;
+    left: -200px;
+    width: 264px;
+    padding-bottom: 12px;
+    display: none;
+  }
+
+  &:hover .tooltip {
```

Comment:
This won't show for non-mouse users. I think it could be pretty trivial to have the tooltip show when the icon is focused. Could you try?